src/registrar/templates/includes/summary_item.html

```diff
@@ -0,0 +1,49 @@
+<section class="summary-item margin-top-3">
```

Comment:
**comment**: I like this simplification very much to make the shared component here.